### DISCUSSION TITLE
Why somtimes the answers were not given based on the input source document?

### DISCUSSION THREAD

**Discussion by sprintcheng** [imartinez/privateGPT#739] [General]
I think model sometimes might try to provide the answers based on its understanding or preset concepts, not strictly follows the imported documentation.

Here is an example, I tried the followings

Here is the content I feed into privateChatGPT, 

https://www.ibm.com/docs/en/filenet-p8-platform/5.5.x?topic=solutions-deadlock-errors-sql-server 

However, when I was asking the following question to privateChatGPT

> Question:
Show me detailed steps to fix MS SQL server deadlock issue in FileNet application.
> Answer (took 181.08 s.):
 To resolve Deadlock issues on Microsoft Server, you can follow these high-level steps: 1) Ensure all servers and clients are shut down; 2) Disable Read/Committed Snapshot Isolation (RCSI); 3) Restart SQL server instance to ensure RCSI is enabled for new connections. 4) Verify XACT_ABORT property on database instance name, if it's checked then disable the feature by running ALTER DATABASE command; 5) Set shared file group attribute of object store tables and objects in FileNet as READONLY=FALSE to enable sharing without any issues during creation

Here I attached is original content from the documentation , where you can see the for point #2 is totally opposite, per the documentation, READ_COMMITTED_SNAPSHOT should be enabled, but per privateChatGPT, first it interpret  READ_COMMITTED_SNAPSHOT as Read/Committed Snapshot Isolation (RCSI), then it mentioned it should be disabled

_Resolving the problem

READ_COMMITTED_SNAPSHOT is required on SQL Server to reduce deadlocks.
**To enable Read-Committed Snapshot Isolation on SQL Server:**

    Shut down all of the servers and clients, and make sure there are no other connections to SQL Server.
    Connect to SQL Server and issue the following command:

    ALTER DATABASE dbName SET READ_COMMITTED_SNAPSHOT ON

Restart SQL Server and verify that the Read-Committed Snapshot Isolation setting takes effect for new connections._

Please help me how to remove or minimize preset model concepts from privateChatGPT.

Thanks
Sprint

**Comment by akashlinux10may** [comment]
same problem I am facing, it answers out the context queries.
